src/pg/sql/clauses.js

```diff
@@ -28,12 +28,24 @@ export const lookup = (prop) => {
     : sql`"${raw(prefix)}"`;
 };
 
+export const lookupNumeric = (prop) => {
+  const [prefix, ...suffix] = encodePath(prop);
+  return suffix.length
+    ? // @ts-ignore sql-template-tag typedef bug
+      sql`CASE WHEN "${raw(
+        prefix,
+      )}" #> ${suffix} = 'null'::jsonb THEN 0 ELSE ("${raw(
+        prefix,
+      )}" #> ${suffix})::numeric END`
+    : sql`"${raw(prefix)}"`;
```

Comment:
We should probably do `COALESCE` in the "else" case, to handle the column value being SQL NULL.

Reply:
I just tried, and I got this 🙏
```
pg.sql_error COALESCE types jsonb and integer cannot be matched;
```